powersimdata/design/investment/README.md

```diff
@@ -1,72 +1,64 @@
-
-
-**dclines: calculate_dc_inv_costs**
+#### dclines: calculate_dc_inv_costs
 
 For all capacity added on a dcline, the investment cost is:
-`Cost ($2015) = Pmax (MW) * lengthMi (miles) * costMWmi ($2015/MW-mi) + dc_term_cost (DC terminal cost $2015)`
+`Cost ($today) = Pmax (MW) * (lengthMi (miles) * costMWmi ($2015/MW-mi) * inflation(2015 to today) + 2 * terminal_cost_per_MW ($2020) * inflation($2020 to today))`
 Then all line costs are summed.
 
-**plant: calculate_gen_inv_costs**
+#### plant: calculate_gen_inv_costs
 
 For all capacity added on a plant, the investment cost is:
-`Cost ($2018) = Pmax (MW) * CAPEX ($2018/MW) * reg_cap_cost_mult (regional capital cost multiplier, unitless)`
+`Cost ($today) = Pmax (MW) * CAPEX ($2018/MW) * reg_cap_cost_mult (regional capital cost multiplier) * inflation($2018 to today)`
 Then all costs are summed by technology (so you can ignore the non-renewables/storage if you want).
 
 
-
 ### Methods
 
-
-
-**branches: calculate_ac_inv_costs**
+#### branches: calculate_ac_inv_costs
 
 - Find new upgrades for each line/transformer (change in MW rating): `grid.branch.rateA - base_grid.branch.rateA`
 - Drop unchanged branches.
 - Map branches to corresponding kV.
 - Separate lines and transformers (TransformerWindings dropped).
 - Lines: Find closest kV present in the corresponding cost table using `select_kv` . Labeled "kV_cost." 
 - Lines: Find MW closest to the line's rateA that exists within the corresponding kV_cost sub-table. Then use that costMWmi base cost value.
-- (messy) Lines: Import bus to NEEM mapping file. Check that no buses you need are missing. Check that buses have the same lat/lon values as the pre-made mapping file. If any issues, re-map NEEM regions to those points.
+- Lines: Import bus to NEEM mapping file. Check that no buses you need are missing. Check that buses have the same lat/lon values as the pre-made mapping file. If any issues, re-map NEEM regions to those points.
 - Lines: Map regional multipliers onto lines' to_bus and from_bus by NEEM region.
 - Lines: Regional multiplier is then the average of the 2 buses' regional multipliers.
+- Lines: Inflation is applied to scale from 2010 dollars to present.
 - Lines: final calculations
-- Xfmrs: Find closest kV present in the corresponding cost table using `select_kv` . Labeled "kV_cost." 
-- Xfmrs: Map kV_cost to corresponding transformer cost.
-
-**Warning**: Now that I think about this again, I bet the transformer costs are too high with this method. If we upgrade the rating just slightly, it's going to count this as an entirely new transformer. We could just drop this investment cost altogether, or just include on totally new transformers? Not sure.
+- Xfmrs: Find closest kV present in the corresponding cost table, for each side of the transformer, to get per-MW values.
+- Xfmrs: Multiply per-MW value by upgraded MW.
+- Xfmrs: Regional multiplier is then applied.
+- Xfmrs: Inflation is applied to scale from 2020 dollars to present.
```

Comment:
Is Xfmrs an abbreviation for transformers? If yes I would use the full word

Reply:
Done